cookbooks/convection-box/tutorial-onset-of-convection/README.md

```diff
@@ -0,0 +1,86 @@
+Author: Lorraine J. Hwang, Ian Rose, Juliane Dannberg and the ASPECT development community
+
+# Introduction to ASPECT
+
+This notebook is based on tutorials by J. Dannberg that provide a basic introduction to ASPECT.
+This notebook demonstrates the onset of convection and the Nusselt-Rayleigh number relationship.
+
+To run, copy the contents of this directory to your workspace.
+
+
+## Running using ASPECT Jupyter Notebooks tool
+
+The current version is verified to run within the ASPECT Jupyter Notebooks tool which can be launched from the CIG website:
+
+https://geodynamics.org/resources/aspectnotebook
+
+
+## Running on your desktop
+
+If you run in your local JupyterLab environment, note the following dependencies:
+
+ * Python 3.8.5
+ * IPython 7.19.0
+ * Jupyterlab 3.2.1
+ * Jupyter widget extension 1.0.0 (see Appendix B)
+ * matplotlib 3.3.2
+ * numpy 1.19.2
+ * tables
+ * ipympl
+ * scipy
+ * glob
+ 
+ 
+## Running using Docker
+The current version of the docker container supports an older version of ASPECT.  The model_input directory contains two files with `_old` in its name which will work with this container.
+
+Use at your own risk.
+
+Here are the old instructions
+
+A. Install Docker
+
+B. Download the docker image
+> docker pull tjhei/aspect-jupyter
```

Comment:
Should we maybe update this image first instead of trying to be compatible with it? This image is from 2017 and contains ASPECT 2.0.0-pre. If we don't update it, we should remove these instructions.

Reply:
The plan is to remove these files and clean-up the README.md file after this pull request is merged.  This way we have a record of it somewhere in the commit history if needed.